Makefile

```diff
@@ -7,6 +7,16 @@ clean :
 	find . -name '*.pyo' -exec rm --force {} +
 	find . -name '*~'    -exec rm --force {} +
 	find . -name '__pycache__'  -exec rm -rd --force {} +
-install:
-	cp siriushlacon/utils/images/LNLS.png ~/.local/share/icons
-	cp miscellaneous/$(NAME).desktop $(DESKTOP)/$(NAME).desktop
+
+install-files:
+	cp siriushlacon/utils/images/LNLS.png /home/sirius/.local/share/icons
```

Comment:
If the package is to be installed for the entire system, the `.png` should be at `/usr/share/icons` 

The name `LNLS.png` is not the most appropriate, maybe in the future rename to `sirius-hla-as-lnls.png` or something like that (i need to refactor the code).

Reply:
Done